### ISSUE TITLE
Default training algorithm: When are the gradients applied?

### ISSUE THREAD

**Issue by renatobellotti** [tensorflow/tensorflow#36244]
## Description of issue (what needs changing):

It is not clear how a Keras model is trained by default. Are the gradients applied after each minibatch, or are they averaged over all minibatches, and then the average gradients are applied?

**Comment by gadagashwini** [comment]
@renatobellotti, can you elaborate the issue with code snippet or any use case. Thanks!

**Comment by gadagashwini** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
